app/main.py

```diff
@@ -35,6 +36,29 @@ class InputString(BaseModel):
 )
 
 
+@app.post("/form/send")
+async def send_form_tweet(data: RequestedFormData):
+    '''
+    Sends a reply tweet with a linked form to gather additional information on an incident.
+
+    Args:
+        data (RequestedFormData):  JSON containing information required to send reply tweet with form link
+            data.tweet_source (str): Full URL to source tweet
+            data.information_requested (str): One of a pre-defined set of information requests:
+                                            - location or date (for now)
+
+            e.g.{
+                    "tweet_source": "https://twitter.com/elonmusk/status/1423830326665650179",
+                    "information_requested": "location"
+                }
+
+    Returns:
+        tweet.id (int): ID of the tweet that was sent
+    '''
```

Comment:
Love this documentation. Tells us exactly what it does, how it works. Something we need more of in our repo. Good job, Brody!

Reply:
Following what Joshua said, great documentation of how this function is intended to work!